proposals/0569-multiline-strings.rst

```diff
@@ -98,6 +98,10 @@ A working prototype is available at `brandonchinn178/string-syntax <https://gith
       * See `Section 2.6 <https://www.haskell.org/onlinereport/haskell2010/haskellch2.html#x7-200002.6>`_ of the Haskell 2010 Report
       * See the example in *Section 3.3 String gaps*
 
+   #. Convert all lexical ``newline`` characters (as defined in the `Haskell report <https://www.haskell.org/onlinereport/haskell2010/haskellch2.html#x7-160002.2>`_) to ``\n``
```

Comment:
I think it's a bit redundant to first *convert* all lexical `newline` characters to `\n`, only to then *split* the string at all such `\n`.

Furthermore, you say `\n` here, but "Split the string by newlines" later. If you going to split by (lexical) newlines anyway, the conversion step seems unnecessary. If you do not mean lexical newline, but rather only `\n`, then perhaps say so.

That said, I would favour the more minimal change that just clarifies the "Split the string by newlines" step and gets rid of this additional conversion step entirely (which seems like an implementation detail at best).

Reply:
Sure, udpated